### ISSUE TITLE
Checkbox on Dashboard Works/Collections pages needs a label

### ISSUE THREAD

**Issue by mjgiarlo** [samvera/hyrax#126]
<a href="https://github.com/jhallida"><img src="https://avatars.githubusercontent.com/u/5582497?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [jhallida](https://github.com/jhallida)**
_Thursday May 05, 2016 at 14:27 GMT_
_Originally opened as https://github.com/projecthydra/sufia/issues/1982_

----

A checkbox hides/shows the sort by and batch edit options in many pages, but this checkbox has no label. Also it is not lined up with the corresponding gear icon.

**Comment by no-reply** [comment]
This is still relevant in 2.x

**Comment by adamjarling** [comment]
@vantuyls @chrisdaaz Any preferences on text for this label?  

![works-show-checkbox-label](https://user-images.githubusercontent.com/3020266/42883548-805af0fe-8a61-11e8-901c-7ed3413f7874.png)

**Comment by adamjarling** [comment]
Here's another visual option of placing the label in front of the dropdown arrow.  Better/worse?

![works-show-checkbox-dropdown-option2](https://user-images.githubusercontent.com/3020266/42884369-c6c103c4-8a63-11e8-83e5-951faec40e66.png)

**Comment by adamjarling** [comment]
After discussing in the Hyrax WG standup today, we'll run with "Select" as the current label...
